Formula/ocrmypdf.rb

```diff
@@ -0,0 +1,99 @@
+class Ocrmypdf < Formula
+  include Language::Python::Virtualenv
+
+  desc "Adds an OCR text layer to scanned PDF files"
+  homepage "https://github.com/jbarlow83/OCRmyPDF"
+  url "https://files.pythonhosted.org/packages/9b/c0/13a0ffb2184b85018b64466d1303ec1ceee05acff0b8ada16fdbf2ec101d/ocrmypdf-5.4.4.tar.gz"
+  sha256 "e523591c5d4e4a8cdeee2e9e24c3f631c8638a803dba74c5b8b274681784dbf4"
+
+  depends_on :python3
+  depends_on "pkg-config" => :build
+  depends_on "tesseract" => :run
+  depends_on "ghostscript" => :run
+  depends_on "qpdf" => :run
+  depends_on "unpaper" => :run
+
+  # For Pillow source install
+  depends_on "openjpeg"
+  depends_on "freetype"
+  depends_on "libpng"
+  depends_on "jpeg"
+  depends_on "webp"
```

Comment:
We should be able to get away without `webp` and `little-cms2` deps too?

Reply:
Removed